### DISCUSSION TITLE
How to make the title, and axis word space larger compared with the curve space

### DISCUSSION THREAD

**Discussion by ytianle** [Profactor/cv-plot#27] [Ideas]
Hi, 

I have a question about how to shrink the size of the plot window and enlarge the space for the title and axis word?

Thank you,
Mark

**Comment by wpalfi** [comment]
Hi Mark,
I guess you want `Axes::setMargins()`? E.g. in https://github.com/Profactor/cv-plot/blob/ec9282f3facc7879e7f2af316c72a214c539d5ae/CvPlot/examples/examples_core.cpp#L119
